commands.go

```diff
@@ -61,7 +61,7 @@ type statefulCmdable struct {
 
 //------------------------------------------------------------------------------
 
-func (c statefulCmdable) Auth(password string) *StatusCmd {
+func (c *statefulCmdable) Auth(password string) *StatusCmd {
```

Comment:
Is this needed? I think `c statefulCmdable` should be faster, because `statefulCmdable` contains only 1 field which is a function pointer. So:
- `c statefulCmdable` - value containing function pointer
- `c *statefulCmdable` - pointer to the value containing function pointer